tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "extends": "./src/config/tsconfig.aegir.json",
     "compilerOptions": {
         "outDir": "dist",
-        "emitDeclarationOnly": true
+        "emitDeclarationOnly": true,
+        "jsx": "preserve"
```

Comment:
aegir build fails without this with the new fixtures, but it shouldn't impact output of aegir since there are no `src/**/*.jsx` or `src/**/*.tsx` files